packages/material-ui-lab/src/AvatarGroup/AvatarGroup.d.ts

```diff
@@ -7,6 +7,10 @@ export interface AvatarGroupProps
    * The avatars to stack.
    */
   children: React.ReactNode;
+  /**
+   * Spacing between avatars
+   */
+  spacing?: 'small' | 'medium' | 'large';
```

Comment:
```suggestion
  spacing?: 'small' | 'medium' | 'large' | number;
```

Reply:
These changes make sense to me, I also updated the proptypes check to reflect the number/string combo